cross/popt/Makefile

```diff
@@ -3,7 +3,9 @@ PKG_VERS = 1.16
 PKG_EXT = tar.gz
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 PKG_DIST_NAME = $(PKG_DIR).$(PKG_EXT)
-PKG_DIST_SITE = http://rpm5.org/files/$(PKG_NAME)
+# rpm5.org has been down since ~2007
+#PKG_DIST_SITE = http://rpm5.org/files/$(PKG_NAME)
```

Comment:
we have a revision system, no need to comment the previous PKG_DIST_SITE
and the comment `rpm5.org has been down since ~2007` should be removed here too. 
But use a commit comment like `fix download location...` to describe the change.

Reply:
:+1: fixed